openlibrary/macros/ShareModal.html

```diff
@@ -38,8 +38,8 @@
               $# rm special characters from url such as single quote which break js prompt/iframe
               $ clean_url = page_url.replace("'", "\\'")
               $ iframe_html = str(embed_iframe(clean_url)).strip()
-              <a class="embed-work-btn" title="$_('Embed this book in your website')" data-ol-link-track="Share|Embed"
-                onclick="prompt('Copy embed code to clipboard:', '$iframe_html');">
+              <a class="embed-work-btn copy-to-clipboard" title="$_('Embed this book in your website')" data-ol-link-track="Share|Embed"
```

Comment:
I think for the embed it is useful show them what they're copying, since there are a few different ways things can be embedded... Ideally I think we'd have maybe a new modal that shows the text with a copy button? ala YouTube. But not a blocker! Just an idea for future. I think this is good as is.

Reply:
Needs design.